src/core/atom_component/dragTree/component/treeNode/SceneTreeNode.re

```diff
@@ -127,26 +120,41 @@ module Method = {
             )
        );
 
-    isTrigger ?
-      DragGameObject(id, startId, dragPosition) :
+    isValid ?
+      DragGameObject(gameObject, startId, dragPosition) :
       isAssetWDBFileFunc() ?
         {
           let wdbGameObjectUid =
             StateEditorService.getState()
             |> OperateTreeAssetEditorService.unsafeFindNodeById(startId)
             |> WDBNodeAssetService.getWDBGameObject;
 
-          DragWDB(id, wdbGameObjectUid, dragPosition);
+          DragWDB(gameObject, wdbGameObjectUid, dragPosition);
         } :
         DragLeave;
   };
 
   let buildNotDragableUl = TreeNodeUtils.buildNotDragableUl;
 
+  let _buildMultipleClassName = (isSelected, isActive, dragGapClass) =>
+    "draggable-container"
+    ++ (
+      isSelected ? isActive ? " select-active " : " select-not-active " : " "
+    )
+    ++ dragGapClass;
```

Comment:
TODO refactor: extract buildMultipleClassName function

Reply:
I already extract it.what you say?